lib/asset/createassetservletpart.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+"use strict";
+
+const TransferPartImport = require("./transferpart");
+const FormData = require("form-data");
+const { HTTP } = require("../constants");
+
+const FILE_OFFSET = "file@Offset";
+const CHUNK_LENGTH = "chunk@Length";
+const FILE_LENGTH = "file@Length";
+const FILE = "file";
+const CHARSET = "_charset_";
+const UTF8 = "utf-8";
+const CHUNKED_CONTENT_TYPE = "x-chunked-content-type";
+const CHUNKED_TOTAL_SIZE = "x-chunked-total-size";
+
+/**
+ * Reflects a unit of content to be transferred, refers to part of an asset. This
+ * part provides functionality specific to uploading to AEM's create asset servlet.
+ */
+class CreateAssetServletPart extends TransferPartImport.TransferPart {
+    /**
```

Comment:
Misses description of what the method does

Reply:
@tmathern At least for VSCode intellisense, leaving the description blank on an overridden class method will inherit the parent method's description. This is why I've left the description missing.

Reply:
Not everyone uses VS Code, so I would suggest to still fill out the doc (at least add that it overrides the method from the parent).